.circleci/config.yml

```diff
@@ -43,30 +39,56 @@ jobs:
       - run:
           name: refresh sources
           command: |
-            rm -r /usr/share/hue/desktop/core/src/desktop
+            if [ ! -d /usr/share/hue ]; then
+              # Ubuntu 20.04 ARM64
+              sudo mkdir -p /usr/share/hue
+              sudo chown -R $USER:$USER /usr/share/hue
+            fi
+
+            rm -rf /usr/share/hue/desktop/core/src/desktop
+            mkdir -p /usr/share/hue/desktop/core/src/
             cp -r desktop/core/src/desktop /usr/share/hue/desktop/core/src/desktop
 
             for lib in `ls desktop/libs`
             do
-              rm -r /usr/share/hue/desktop/libs/$lib/src/$lib
+              rm -rf /usr/share/hue/desktop/libs/$lib/src/$lib
+              mkdir -p /usr/share/hue/desktop/libs/$lib/src/
               cp -r desktop/libs/$lib/src/$lib /usr/share/hue/desktop/libs/$lib/src/$lib
             done
-            for lib in `ls apps | grep -v Makefile`
+            for lib in `ls apps | grep -v Makefile | grep -v logs`
```

Comment:
excluding the `logs` folder is needed to be able to run `circleci` cmdline tool locally